# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/operation/vertex_state/correctness/cts.https.html.ini
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/operation/vertex_state/correctness/cts.https.html.ini
--- a/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/operation/vertex_state/correctness/cts.https.html.ini	Fri Oct 03 18:01:30 2025 +0000
+++ b/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/operation/vertex_state/correctness/cts.https.html.ini	Fri Oct 03 18:08:29 2025 +0000
@@ -1,174 +1,95 @@
 [cts.https.html?q=webgpu:api,operation,vertex_state,correctness:array_stride_zero:*]
   implementation-status:
     if os == "linux": backlog
-    if os == "mac": backlog
   [:format="float16"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float16x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float16x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float32"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float32x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float32x3"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="float32x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint16"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint16x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint16x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint32"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint32x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint32x3"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint32x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint8"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint8x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="sint8x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="snorm16"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="snorm16x2"]
     expected:
       if os == "linux": FAIL
-      if os == "mac": FAIL
 
   [:format="snorm16x4"]
     expected:
       if os == "linux": FAIL
-      if os == "mac": FAIL
 
   [:format="snorm8"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="snorm8x2"]
     expected:
       if os == "linux": FAIL
-      if os == "mac": FAIL
 
   [:format="snorm8x4"]
     expected:
       if os == "linux": FAIL
-      if os == "mac": FAIL
 
   [:format="uint16"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint16x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint16x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint32"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint32x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint32x3"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint32x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint8"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint8x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="uint8x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm10-10-10-2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm16"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm16x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm16x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm8"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm8x2"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm8x4"]
-    expected:
-      if os == "mac": FAIL
 
   [:format="unorm8x4-bgra"]
-    expected:
-      if os == "mac": FAIL
 
 
 [cts.https.html?q=webgpu:api,operation,vertex_state,correctness:buffers_with_varying_step_mode:*]